gensim/scripts/word2vec2tensor.py

```diff
@@ -67,8 +68,8 @@ def word2vec2tensor(word2vec_model_path, tensor_filename, binary=False):
     outfiletsv = tensor_filename + '_tensor.tsv'
     outfiletsvmeta = tensor_filename + '_metadata.tsv'
 
-    with open(outfiletsv, 'w') as file_vector:
-        with open(outfiletsvmeta, 'wb') as file_metadata:
+    with smart_open(outfiletsv, 'w') as file_vector:
```

Comment:
No `w` -- always use binary I/O (`rb`, `wb`). Encode text to utf8 where necessary.

Reply:
Changing that w to wb gives me the following error: 

Traceback (most recent call last):
  File "word2vec2tensor.py", line 94, in <module>
    word2vec2tensor(args.input, args.output, args.binary)
  File "word2vec2tensor.py", line 76, in word2vec2tensor
    file_vector.write(vector_row + '\n')
TypeError: a bytes-like object is required, not 'str'

Reply:
Yes -- please encode text to binary (utf8) where necessary.